src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -166,6 +166,8 @@ public void EnsureMetadataVirtual()
 
         private OverriddenOrHiddenMembersResult _lazyOverriddenOrHiddenMembers;
 
+        protected bool _localsAreZeroed;
```

Comment:
Protected members should not begin with an `_`. Really protected members should be Pascal cased but somehow other members here are camelCased. Lots of bad examples.

Reply:
Actually, I'll set it to private. I just realized there's no benefit in it being protected.